README.md

```diff
@@ -19,9 +19,9 @@ when you add the following to `package.json`:
 
 ```js
   "browserslist": [
-    "last 1 version",
-    "> 1%",
-    "maintained node versions",
+    "last 2 versions",
```

Comment:
This query is very close  to `defaults`. Maybe we can promote `defaults` here? For instance:

```
defaults
not IE 11
maintained node versions
```